control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -1618,6 +1618,15 @@ func (r *HostedControlPlaneReconciler) reconcileKubeAPIServer(ctx context.Contex
 		return fmt.Errorf("failed to reconcile authentication token webhook config: %w", err)
 	}
 
+	pdb := manifests.KASPodDisruptionBudget(hcp.Namespace)
+	if result, err := controllerutil.CreateOrUpdate(ctx, r, pdb, func() error {
+		return kas.ReconcilePodDisruptionBudget(pdb, p)
+	}); err != nil {
+		return fmt.Errorf("failed to reconcile api server pdb: %w", err)
+	} else {
+		r.Log.Info("reconciled api server pdb", "result", result)
```

Comment:
nit: Log should start with upper level by convention. Also aside from this PR it'd be nice to set on this pattern consistently across all calls to createOrUpdate. Seem sometimes we are logging others we aren't arbitrarily.

Reply:
Fixed